editor/test/integration/extension_spine_test.clj

```diff
@@ -12,7 +12,7 @@
 ;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
 ;; specific language governing permissions and limitations under the License.
 
-(ns integration.spine-extension-test
+(ns integration.extension-spine-test
```

Comment:
Renamed this test for consistency so it corresponds to the name of `extension-spine`.